src/pybind/mgr/telemetry/tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
 envlist =
     py3
-    mypy
```

Comment:
are we removing this because the tox.ini in pybind/mgr should cover checking the telemetry dir's typing?

Reply:
this was discussed offline. This instance of `mypy` had no effect as no `mypy` env exists in this tox.ini file